forms-flow-web/src/components/Draft/DraftTable.js

```diff
@@ -77,24 +84,34 @@ const DraftTable = () => {
 
 
   const handlePageChange = (page) => {
+    dispatch(setDraftListLoading(true));
     dispatch(setDraftListActivePage(page));
   };
 
   const onSizePerPageChange = (limit) => {
+    dispatch(setDraftListLoading(true));
     setPageLimit(limit);
     dispatch(setCountPerpage(limit));
     dispatch(setDraftListActivePage(1));
   };
 
+  const updateSort = (sortOrder,sortBy) => {
+    dispatch(setDraftListLoading(true));
+    dispatch(setDraftSortOrder(sortOrder));
+    dispatch(setDraftSortBy(sortBy));
+    dispatch(setDraftListActivePage(1));
+  };
+
   return (
     <>
       <div style={{ minHeight: "400px" }}>
+      <LoadingOverlay active={isDraftLoading} spinner text={t("Loading...")}>
         <table className="table custom-table table-responsive-sm">
           <thead>
             <tr>
-              <th>{t("Draft Id")}</th>
-              <th>{t("Draft Title")}</th>
-              <th>{t("Last Modified")}</th>
+              <th>{t("Id")} {isAscending && sortBy === 'id' ? <i  onClick={() => updateSort('desc','id')} className="fa-sharp fa-solid fa-arrow-down-9-1" /> :  <i onClick={() => updateSort('asc','id')} className="fa-sharp fa-solid fa-arrow-up-1-9" />}</th>
+              <th>{t("Title")}{isAscending && sortBy === 'DraftName' ? <i onClick={() =>updateSort('desc','DraftName')} className="fa-sharp fa-solid fa-arrow-down-a-z"/> : <i onClick={() =>updateSort('asc','DraftName')}   className="fa-sharp fa-solid fa-arrow-up-z-a"/>}</th>
```

Comment:
`
const generateSortableHeader = (label, sortKey, currentSortBy, isAscending, updateSort) => {
  const ascendingIcon = isAscending && currentSortBy === sortKey ? "fa-sharp fa-solid fa-arrow-down-9-1" : "fa-sharp fa-solid fa-arrow-up-1-9";
  const descendingIcon = isAscending && currentSortBy === sortKey ? "fa-sharp fa-solid fa-arrow-down-a-z" : "fa-sharp fa-solid fa-arrow-up-z-a";
  return (
    <th>
      {label}
      {isAscending && currentSortBy === sortKey ? (
        <i onClick={() => updateSort('desc', sortKey)} className={`fa-sharp fa-solid ${descendingIcon}`} />
      ) : (
        <i onClick={() => updateSort('asc', sortKey)} className={`fa-sharp fa-solid ${ascendingIcon}`} />
      )}
    </th>
  );
};
`





### Use the function
`{generateSortableHeader(t("Id"), 'id', sortBy, isAscending, updateSort)}
{generateSortableHeader(t("Title"), 'DraftName', sortBy, isAscending, updateSort)}
{generateSortableHeader(t("Last Modified"), 'modified', sortBy, isAscending, updateSort)}
`